perma_web/frontend/lib/store.js

```diff
@@ -0,0 +1,25 @@
+import { globalStore } from '../stores/globalStore'
+
+export const getLinksRemainingStatus = (links_remaining, is_nonpaying) => {
+    if (links_remaining !== 'Infinity') {
```

Comment:
Since `links_remaining = Number({{ links_remaining }})`, I think that means this is not the string `'Infinity'` here, but rather, the JS Infinity object. Do you agree?

![image](https://github.com/harvard-lil/perma/assets/11020492/29c4bc8c-48f2-40f3-8a41-41a3e1f530bf)

Reply:
Yes, you are 100% correct and this totally causes a visual bug if not fixed 😮‍💨 where a user will see a link counter that says that they have "Unlimited" links remaining because they are marked as a metered, not unlimited user